python/DileptonPreselector_cfi.py

```diff
@@ -2,7 +2,7 @@
 
 dileptonPreseletor = cms.EDFilter("DileptonPreselector",
 	muons = cms.InputTag("slimmedMuons"),
-	nMuons = cms.double(1),
+	nMuons = cms.double(0),
```

Comment:
As we have discussed, the `*_cfi.py` files **should not be changed**. Please revert.